### PR TITLE
Publish paola

### PR DESCRIPTION
**Pull request by paolap**
I tried to improve the issue with the publish-options page looking bad

1) I moved Zenodo, Figshare.. out of CLEX and gave it its own "generalist repositories" title. I added a sentence at the start of the page to clarify this.

2) I created tabs for each group: BoM, CSIRO, CLEX, Generalist repositories and Discipline repositories.
Example with TABS

3) Move ESGF after NCI-geonetwork

4) Updated publish-intro as the text didn't match anymore the actual guidelines

Apart from these I also edited the publish-zenodo page

Finally this is a question:
We have in the book appendix a list of journal editors and their publishing requirements, we should somehow link them to this guidelines. 
As we also have a page with dataset examples at the end of the guidelines, we should be consistent. Either put both in the book appendix or create a kind of guideline appendix and move the journals requirements at the end of the guidelines too
Any idea?

### CODE REVIEW

**Review by AliciaTak — APPROVED** [review]
It looks good to me Paola. I haven't had a detailed read of the book, but it makes sense to move the dataset examples to the appendix, too.

File: Governance/publish/publish-intro.md [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
-This guidelines are covering the main steps involved in publishing a climate dataset.
-They illustrate how to prepare a dataset for publication, what you want to include both in the dataset itself and in the metadata that will be created for the publication. What are the common requirement for a dataset to publication ready and what are th specific requirements based on the publisher. As this is looking at the Australian context we focused on the institutions we know of that publish climate data. If you are aware of others we didn't covert please contact us by creating an issue.
+These guidelines are covering the main steps involved in publishing a climate dataset.
+We tried to list the publishing options available to climate researchers in Australia, based on their organisation and the type of data to publish. 
+We then covered some of these options in detail, highlighting how to prepare a dataset for publication, what to include both in the dataset itself and in the metadata record.
+We focused on institutions and projects we know of that publish climate data. If you are aware of others that we didn't cover please contact us by creating an issue.
```
Comment (hot007) [inline Governance/publish/publish-intro.md:6]: link [creating an issue](https://github.com/ACDguide/Governance/issues/new/choose) in this sentence.

File: Governance/publish/publish-options.md [hunk 2]
```diff
@@ -2,41 +2,55 @@
...
-In some cases, there might be the option of contributing to a large-scale project that has its own publishing procedure established.
+Another important consideration is the kind of data, depending on format, size etc. using a public repository might be suitable and at time easier.
+In some cases, depending on the dataset, using a public repository is suitable and often the easiest option. ori
+On the other end of the scale, some data could be produced on purpouse or simply be suitable to contribute to a large-scale project that has its own publishing procedure established.
 Here we are covering the most common use cases for our community.
 
-## BoM
+::::{tab-set}
+:::{tab-item} BoM 
+Insert here pathways for BoM researchers...
```
Comment (hot007) [inline Governance/publish/publish-options.md:12]: Request review/contribution from someone at BoM - who?

**Review by hot007 — APPROVED** [review]
Looks good @paolap , nice re-ordering. I made one small edit and suggested addition of a link to create issues.

Regarding you question about page/appendix consistency, I actually wonder if that page of journal requirements isn't at least as relevant as the exemplar datasets page in this chapter - I would support moving the journal requirements page from appendices to this chapter, depending what you and others think?

File: Governance/publish/publish-intro.md (hunk 1 above)
Comment (paolap) [reply Governance/publish/publish-intro.md:6]: about the journal/dataset examples I agreed they should either both in the appendix or at the end of this chapter, which might make more sense, but they're basically equally relevant. And I'll add the link, thanks for spotting that

Comment (hot007) [reply Governance/publish/publish-intro.md:6]: Yep I'd be happy for it to be moved in here (ie update toc), feel free to add it to this PR if possible.

File: Governance/publish/publish-options.md (hunk 2 above)
Comment (paolap) [reply Governance/publish/publish-options.md:12]: I think Avi was happy to contribute and Griffith (?) mentioned they're publishing with C3S so he could contribute that

**Comment by paolap** [comment]
I've added the issue link requested by Claire plus a few more for the portals listed in publish-options (as TERN, CDS etc). I've also moved the journal list from the appendix to these guidelines. Can easily be moved back. I don't think anything in here breaks the main and two people reviewed it so @chloemackallah if you're busy it's still probably safe to merge it. Should you hate the restructure it's not too difficult to revert.

**Review by chloemackallah — APPROVED** [review]
Looks great, I like the tabs a lot!

**Pull request merged**